oeq-ts-rest-api/src/Search.ts

```diff
@@ -1,6 +1,6 @@
 import * as Common from './Common';
-import { is } from 'typescript-is';
-import { GET } from './AxiosInstance';
+import {is} from 'typescript-is';
+import {GET} from './AxiosInstance';
```

Comment:
It looks like these got missed by prettier
```suggestion
import { is } from 'typescript-is';
import { GET } from './AxiosInstance';
```